pkg/dyninst/decode/json_schema.go

```diff
@@ -0,0 +1,128 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package decode
+
+import (
+	"github.com/google/uuid"
+)
+
```

Comment:
This file is going to go away, i'm just using it as reference.

Reply:
Ack, left my comment below before seeing this.